### DISCUSSION TITLE
Image Feed

### DISCUSSION THREAD

**Discussion by JoeGay777** [melMass/comfy_mtb#135] [Q&A] [answered]
Is there a way to disable the image feed at the bottom of the screen?

Thanks.

**Comment by melMass** [comment, marked as answer]
Yes, you can click on the icon ( 🖼️) in the top of the floating panel, I actually forgot I was still shipping it tbh since I use [this](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/tree/main?tab=readme-ov-file#image-feed) now and my extension disables image feed if pythongoss's image feed is already installed, I will add a way to enable or disable js extensions (tracked in #136)

**Reply by JoeGay777** [reply]
Thank you.
